doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,17 @@
 
 <h3>New features since last release</h3>
 
+* Parametric operations now have the `parameter_frequencies`
+  method that returns the frequencies with which a parameter
+  enters a circuit when using the operation.
+  [(#2180)](https://github.com/PennyLaneAI/pennylane/pull/2180)
+
+  The frequencies can be used for circuit analysis, optimization
+  via the `RotosolveOptimizer` and differentiation with the
+  parameter-shift rule. They assume that the circuit returns
+  expectation values or probabilities, for a variance
+  measurement the frequencies will differ.
```

Comment:
Still feels like this should be part of the name somehow, but I can never think of anything better than `parameter_frequencies` 😆 I try to have a 'zero or one underscore' rule when naming

Reply:
Yeah, I was thinking about this as well, but I'd say it is okay, because the whole fourier module and Rotosolve also assume this context of expectation value/probs outputs :)